source/requirements.txt

```diff
@@ -9,7 +9,7 @@ marshmallow==3.14.0
 marshmallow-sqlalchemy==0.26.1
 gunicorn==20.1.0
 psycopg2-binary==2.9.1
-pymisp==2.4.119
+pymisp==2.4.152
```

Comment:
I think we can just remove this one now, since the requirement is included in the wheel package of iris-misp-module

Reply:
Fixed with https://github.com/dfir-iris/iris-web/commit/31fcaaa83f91bd8bbf490e89f4096b4c6cc12a0c